core/vmutils/src/main/java/org/arakhne/afc/vmutil/MACNumber.java

```diff
@@ -200,13 +200,9 @@ public static String join(MACNumber... addresses) {
 				NetworkInterface inter;
 				while (interfaces.hasMoreElements()) {
 					inter = interfaces.nextElement();
-					try {
-						final byte[] addr = inter.getHardwareAddress();
-						if (addr != null) {
-							av.add(new MACNumber(addr));
-						}
-					} catch (SocketException exception) {
-						//
+					final byte[] addr = inter.getHardwareAddress();
```

Comment:
This replacement avoids to search for a next valid address (with the while look).